model/project_ref.go

```diff
@@ -71,6 +71,8 @@ type ProjectRef struct {
 
 	Triggers       []TriggerDefinition       `bson:"triggers,omitempty" json:"triggers,omitempty"`
 	PeriodicBuilds []PeriodicBuildDefinition `bson:"periodic_builds,omitempty" json:"periodic_builds,omitempty"`
+	// List of commands
+	WorkstationConfig []WorkstationCommand `bson:"workstation_config,omitempty" json:"workstation_config,omitempty"`
```

Comment:
I think this should look more like the following. The include modules can be omitted for a future ticket, but I think this gives us room to grow: 
```
type WorkstationProjectConfig {
IncludeGitClone bool
IncludeModules []struct{org, repo, branch, path, string} 
Commands []WorkstatationProjectSetupCommand
}
```

Reply:
I'm fine putting things in a Config to give us room to grow. I'm going to put IncludeGitClone into the commands object though, since order matters.

Reply:
I think we shouldn't give people the control of ordering that. always do it first.

Reply:
🤷 but why? it's not hard to add to the ordering. What if they want to make a directory first or something?

Reply:
because I think that if we assume that everything is in repos, and we don't allow them to do weird things? I mean we could, but I also don't want to change the schema of workstation command, because that gets harder to validate?